rules/internal/bincapz.yara

```diff
@@ -3,9 +3,14 @@ rule bincapz : harmless {
 		__bincapz__ = "true"
 		description = "match the bincapz binary, to omit it unless --ignore-self=false"
     strings:
-        $report = "bincapz/pkg/report"
-		$behavior = "bincapz.Behavior"
-		$render = "bincapz/pkg/render"
+		$b_behavior = "bincapz.Behavior"
+		$f_action = "bincapz/pkg/action"
+		$f_bincapz = "bincapz/pkg/bincapz"
+		$f_compile = "bincapz/pkg/compile"
+		$f_profile = "bincapz/pkg/profile"
+		$f_render = "bincapz/pkg/render"
+        $f_report = "bincapz/pkg/report"
+        $f_version = "bincapz/pkg/version"
     condition:
-        2 of them
+        $b_behavior and 2 of ($f_*)
```

Comment:
Might make sense to do `all of ($f_*)`?

Reply:
Updated in [`80f7ed1` (#323)](https://github.com/chainguard-dev/bincapz/pull/323/commits/80f7ed166eefce38f72557f101b24222c396f6a0).